actix-utils/CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 * Upgrade `tokio-util` to `0.3`.
 * Remove unsound custom Cell and use `std::cell::RefCell` instead, as well as `actix-service`.
+* Provide correctly spelled `LocalWaker::is_registered` method and deprecate the replaced.
```

Comment:
"deprecate" here is not correct for me, I presume it's an oversight in e5b1aebc50d4fd185a71895b450beefbecb7d907.

Reply:
ahh good spot, yeah i had the change in to keep the old version but realised that we had breaking changes anyway